packages/flutter/lib/src/rendering/sliver_grid.dart

```diff
@@ -348,14 +355,17 @@ class SliverGridDelegateWithMaxCrossAxisExtent extends SliverGridDelegate {
   /// cross-axis extent.
   ///
   /// All of the arguments must not be null. The `maxCrossAxisExtent` and
-  /// `crossAxisSpacing` arguments must not be negative. The `crossAxisCount`
-  /// and `childAspectRatio` arguments must be greater than zero.
+  /// `mainAxisSpacing`, and `crossAxisSpacing` arguments must not be negative.
```

Comment:
You can use `[` rather than backticks for these because they map to properties of this object.